### DISCUSSION TITLE
None of the icons are showing up in default install of LazyVim

### DISCUSSION THREAD

**Discussion by jacklund** [LazyVim/LazyVim#2730] [Q&A] [answered]
I'm trying out LazyVim, and followed the installation instructions, haven't modified any configs (yet). However, none of the icons show up at all. Examples:

![image](https://github.com/LazyVim/LazyVim/assets/209634/b1a68031-5852-48f3-9448-7ec768052e9e)

![image](https://github.com/LazyVim/LazyVim/assets/209634/ae6ab6c2-7efb-4372-a5fc-12b1c80ea4af)

I've installed this on both Debian 11.9 and Ubuntu 22.04, with nvim 0.9.

I've looked through the docs, but it doesn't say anything about having to install separate fonts or anything. The only help I can find online is to install NerdFonts, but they don't mention which fonts to use. I've tried installing random ones on my machine but the issue persists.

Help would be greatly appreciated!!

**Comment by dpetka2001** [comment, marked as answer]
[LazyVim Requirements](https://www.lazyvim.org/#%EF%B8%8F-requirements) specifically mentions
> a [Nerd Font](https://www.nerdfonts.com/)(v3.0 or greater) (optional, but needed to display some icons)

You also have to configure your terminal to use the fonts you installed.

**Reply by hems** [reply]
there are tons of Nerd Fonts in here:
https://www.nerdfonts.com/font-downloads

i tried installing symbols-only, but it didn't do the job.

**Reply by gpbPiazza** [reply]
Just sharing info to help, other people if this kind of issue

```bash
echo -e "\ue7a8 \uf09b \uf303 \ue795"
```

If you se the icons you have a nerd font installed in you terminal, if not you have to install

**Comment by jacklund** [comment]
Wow! Thanks for the quick response!

> [LazyVim Requirements](https://www.lazyvim.org/#%EF%B8%8F-requirements) specifically mentions
>
> > a [Nerd Font](https://www.nerdfonts.com/)(v3.0 or greater) (optional, but needed to display some icons)
>
> You also have to configure your terminal to use the fonts you installed.

Ugh, apologies, my brain interpreted "configure your terminal" as "install in the host" - since I'm using a remote terminal, that's probably not going to cut it. 🤦‍♂️ 

Apologies again, and thanks again for the quick response

**Comment by tf42src** [comment]
If anyone else lands here and is looking for **mac + iTerm2**

Go to settings and enable this (after installing the font: double click and press install):
![Screenshot 2024-06-25 at 6 05 30 PM](https://github.com/LazyVim/LazyVim/assets/9712861/e5268ebb-2105-4017-8e16-e538e4f137ac)

**Reply by hems** [reply]
this did the trick! 💘

**Reply by 01ChenQing** [reply]
<img width="1059" height="622" alt="Screenshot 2025-08-30 at 2 31 49 PM" src="https://github.com/user-attachments/assets/890f3fbb-27dd-4f10-acc0-dbab6dddde0e" />
